docs/user-guide/environment-variables.md

```diff
@@ -43,12 +45,21 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 | SD_TEMPLATE_VERSION | Version of the template the job is using (blank if not using template)|
 | SD_TOKEN | JWT token for the build |
 
+## Plugins
+These environment variables may or may not be available depending on what plugins are installed.
+
+#### Coverage (Sonar)
+| Name | Value |
+|------|-------|
+| SD_SONAR_AUTH_URL | Screwdriver API endpoint for authenticating against Sonar |
```

Comment:
Shall we make it clear that this endpoint is to give you an access token against Sonar? By reading this line i will feel this is just like a login stuff